build/config.sh

```diff
@@ -7,7 +7,6 @@ pushd $BUILD_DIR > /dev/null
 export GOVERSION=${GOVERSION:-1.10.3}
 export REVISION=${GIT_COMMIT:-$(git rev-parse HEAD)}
 export LIBRARY_REVISION=$(cat Gopkg.lock | grep github.com/phrase/phraseapp-go -A 2 | tail -n 1 | cut -d '"' -f 2)
-export CURRENT_DATE=$(TZ=UTC date +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
introduce `CHANGE_DATE` with last commit as default
`export CHANGE_DATE=${SOURCE_DATE_EPOCH:- $(git show HEAD --format=format:'%ct')}`

Reply:
Using the date of the last git commit is a good idea